core/controllers/admin_test.py

```diff
@@ -249,6 +275,23 @@ def test_load_new_structures_data(self):
         self.assertEqual(len(questions), 3)
         self.logout()
 
+    def test_load_dummy_skill_data(self):
```

Comment:
Please change "load" to "generate", here and elsewhere.

Reply:
Done.